core/src/test/java/hudson/MarkupTextTest.java

```diff
@@ -102,7 +112,8 @@ public void testPreEscape() {
     }
 
     /* @Bug(6252) */
```

Comment:
Sure, just a note for any future PRs you want to do. (Thanks for this work BTW!)